xray/services/scan.go

```diff
@@ -111,6 +112,9 @@ func (ss *ScanService) ScanGraph(scanParams XrayGraphScanParams) (string, error)
 			return "", fmt.Errorf("failed sending Git Info to XSC service, error: %s ", err.Error())
 		}
 		scanParams.MultiScanId = multiScanId
```

Comment:
is it OK to pass it as param and environment variable as well?

Reply:
Removed all places we use multi_scan_id param